tor/core/inbox.py

```diff
@@ -63,8 +63,8 @@ def process_mod_intervention(post, config):
         # Nothing offensive here, why did this function get triggered?
         return
 
-    # Wrap each phrase in double-quotes (") and commas in between
-    phrases = '"' + '", "'.join(phrases) + '"'
+    # Wrap each phrase in double-quotes (") and commas in between.
+    phrases = f'"{", ".join(phrases)}"'
```

Comment:
This does not do the same thing.
```
>>> phrases = ['asdf', 'qwer', 'erty']
>>> '"' + '", "'.join(phrases) + '"'
'"asdf", "qwer", "erty"'
>>> f'"{", ".join(phrases)}"'
'"asdf, qwer, erty"'
>>>
```

Reply:
Additionally, it might work better to iterate in a different way:

```python
phrases = ', '.join([f'"{phrase}"' for phrase in phrases])
```

This way we have wrapping of each phrase in the list in double-quotes, then join it by commas. Two separate actions.